src/PresentationalComponents/Email/Email.js

```diff
@@ -93,6 +83,19 @@ const Email = () => {
         console.log('cancel pressed');
```

Comment:
console log :eyes:

Reply:
Yeah, filip is working on a feature to make the buton alive.